app/views/ConfirmationView.scala.html

```diff
@@ -31,13 +33,19 @@
         breakdownHelper: ConfirmationBreakdownHelper
 )
 
-@(grant: Grant, regularPay: BigDecimal, version: String)(implicit request: Request[_], messages: Messages)
+@(grant: Grant, calculatorVersion: String)(implicit request: Request[_], messages: Messages)
+
+    @now = @{
+        val formatter = DateTimeFormatter.ofPattern("dd MMMM yyyy")
+        LocalDate.now().format(formatter).toString
```

Comment:
you can use ViewUtils.datetoString(...)

Reply:
yeah eventually did - that needs to be removed.